### PR TITLE
TW-621: add ability to buy nft from collectibles section for alien

### PR DESCRIPTION
**Pull request by Boltman92**

### CODE REVIEW

File: src/screens/collection/components/collectible-item.tsx [hunk 1]
```diff
@@ -48,9 +50,22 @@ export const CollectibleItem: FC<Props> = memo(({ item, collectionContract, sele
   const quantityByUser = item?.holders?.find(holder => holder.holder_address === selectedPublicKeyHash)?.quantity ?? 0;
 
   const isAbleToList = quantityByUser > listedByUser;
+  const isListed = item.listed !== undefined && item.listed !== 0;
```
Comment (lendihop) [inline src/screens/collection/components/collectible-item.tsx:53]: isDefined

Comment (lendihop) [reply src/screens/collection/components/collectible-item.tsx:53]: mb rename to listedAmount?

Comment (Boltman92) [reply src/screens/collection/components/collectible-item.tsx:53]: isDefined doesn't check if we receive 0

Comment (lendihop) [reply src/screens/collection/components/collectible-item.tsx:53]: replace just "item.listed !== undefined"

**Review by lendihop — APPROVED** [review]

**Review by svjatoslavtt — APPROVED** [review]

**Review by lendihop — APPROVED** [review]

**Review by IrynaKhyzhynska — APPROVED** [review]
android done 1.15_763

**Review by vladvn1905 — APPROVED** [review]
ios done build: 1.15_763

**Pull request merged**